### PR TITLE
Addition of 80405 Game Theory

### PR DESCRIPTION
**Pull request by JCao-CMU**
# Summary of Changes

- Please indicate the classes you are modifying:
  - 80-405 （addition of new class）

# Checklist:

Make sure that your contributions:

- [ √] Are accurate and will not mislead the reader
- [√ ] Do not encourage or cause academic integrity violations
- [√ ] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: electives/80405.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+# 80-405: Game Theory
+---
+| Category | Difficulty |
+| -------- | ---------- |
+| Homeworks |   3     |
+
+
+Game theory is a philosophy department course that served as the introduction to basic game theory. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, and sequantial games. Although the class informally assumes some knowledge about decision theory, it is a introductory class, and all of the decision theory part is easy to understand, as it is not in-depth.
```
Comment (mikinty) [inline electives/80405.md]: "an introductory"

Comment (mikinty) [reply electives/80405.md]: Are there math prereqs for understanding the course?

File: electives/80405.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+| Category | Difficulty |
+| -------- | ---------- |
+| Homeworks |   3     |
+
+
+Game theory is a philosophy department course that served as the introduction to basic game theory. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, and sequantial games. Although the class informally assumes some knowledge about decision theory, it is a introductory class, and all of the decision theory part is easy to understand, as it is not in-depth. 
+
+## Lectures
+The lecture besically covers everything about the course material, and attending lectures would be sufficient to complete all homeworks. Lecture notes are provided beforehand (and enouraged to read before the lecture), but those are not in-depth explanations of the course materals bur rather concepts the lecture is going to use to solve some in-class examplar problems. At the last 60-30 minutes of the lecture, TA will typically take over and students will form small groups to discuss the model presented in the lecture.
```
Comment (mikinty) [inline electives/80405.md]: *but not bur

Comment (mikinty) [reply electives/80405.md]: 30-60 is more colloquial than 60-30

File: electives/80405.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Homeworks
+Homeworks are the only element that count as the grade in this course. Yes, this course do not have exams. Homeworks is typically composed of 3-4 problems that covers the topic of interest, and it is often a variant of the problems we solved in the class. There are about 5 homework throughout the semester, and each homework is equally distributed.
```
Comment (mikinty) [inline electives/80405.md]: Is the grading for homework harsh or easy?

Comment (mikinty) [inline electives/80405.md]: "5 homework [assignments]..."

File: electives/80405.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Lectures
+The lecture besically covers everything about the course material, and attending lectures would be sufficient to complete all homeworks. Lecture notes are provided beforehand (and enouraged to read before the lecture), but those are not in-depth explanations of the course materals bur rather concepts the lecture is going to use to solve some in-class examplar problems. At the last 60-30 minutes of the lecture, TA will typically take over and students will form small groups to discuss the model presented in the lecture.
+
+## Homeworks
+Homeworks are the only element that count as the grade in this course. Yes, this course do not have exams. Homeworks is typically composed of 3-4 problems that covers the topic of interest, and it is often a variant of the problems we solved in the class. There are about 5 homework throughout the semester, and each homework is equally distributed. 
+
+## Textbooks
```
Comment (mikinty) [inline electives/80405.md:18]: *especially because it is assigned

File: electives/80405.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+The textbook of this class is Osborne's **An Introduction to Game Theory**, which you can find in many sources. The lecture was build around the content of this book, and reading this book can be very helpful, expecially it is oftentimes assigned reading. If you are really interested at game theory, this book is going to be useful even after the class.
```
Comment (mikinty) [inline electives/80405.md]: *interested about game theory

**Review by mikinty — CHANGES_REQUESTED** [review]
Overall is fine. Would like some more details about justification of difficulty, and grammar fixes.

**Comment by JCao-CMU** [comment]
Addressed. Please check the newly edited version.

**Comment by mikinty** [comment]
> Addressed. Please check the newly edited version.

"Expecially" is still misspelled in the last paragraph. See the comment I suggested

**Comment by JCao-CMU** [comment]
Grammar and typo fixed

File: electives/80405.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+## Homeworks
+Homeworks are the only element that counts as the grade in this course. Yes, this course does not have exams. Homeworks is typically composed of 3-4 problems that cover the topic of interest, and it is often a variant of the problems we solved in the class. There are about 5 homework assignments throughout the semester, and each homework is equally weighted and spaced throughout the semester. Each homework assignment typically spans a week and a half, and extensions can be granted once the professor is noticed 24 hours before the deadline. The grading of the homework is dependent on the TA grader. At the beginning of the semester, each small student group will have a TA assigned, and that TA will be responsible for all questions and all gradings. From previous course data, graders were relatively lenient.
+## Textbooks
+The textbook of this class is Osborne's **An Introduction to Game Theory**, which you can find in many sources. The lecture was built around the content of this book, and reading this book can be very helpful, especially it is oftentimes assigned reading. If you are interested in game theory, this book is going to be useful even after the class.
```
Comment (mikinty) [inline electives/80405.md]:
```suggestion
The textbook of this class is Osborne's **An Introduction to Game Theory**, which you can find in many sources. The lecture was built around the content of this book, and reading this book can be very helpful, especially because it is oftentimes assigned reading. If you are interested in game theory, this book is going to be useful even after the class.
```

File: electives/80405.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (mikinty) [inline electives/80405.md]:
```suggestion
```

Comment (mikinty) [reply electives/80405.md]: extra space

File: electives/80405.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+# 80-405: Game Theory
+---
```
Comment (mikinty) [inline electives/80405.md]: Why is this here?

File: electives/80405.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+| Category | Difficulty |
+| -------- | ---------- |
+| Homeworks |   3     |
+| Exams |   None   |
+
+Game theory is a philosophy department course that served as an introductory to basic game theories. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, prisoner's dilemma, and sequential games. Although the class informally assumes some knowledge about decision theory, it is an introductory class, and all of the decision theory parts are easy to understand, as it is not in-depth. There are no prerequisites for this class, nor is there a requirement for a solid mathematic background.
+## Lectures
```
Comment (mikinty) [inline electives/80405.md:10]:
```suggestion

## Lectures

```

Comment (mikinty) [reply electives/80405.md:10]: Generally we add spaces above and below headers in Markdown

**Review by mikinty — CHANGES_REQUESTED** [review]
Please see comments

It may be helpful to install a Markdown linter if you can to fix style

**Comment by mikinty** [comment]
I'm going to let this pass, but in the future can you please squash commits or make more meaningful changes per commit? Seeing single line changes split into many commits is not helpful, and just pollutes the version history.

You should also be providing more helpful commit messages other than just `Update <file>`

File: electives/80405.md [hunk 10]
```diff
@@ -7,7 +7,9 @@
 | Exams |   None   |
 
 Game theory is a philosophy department course that served as an introductory to basic game theories. The main topics include Minimax Theory, Nash Equilibrium, Sequential Games, Bayesian Games, and Bargaining Theory. Scenarios include zero-sum games, auctions, votes, prisoner's dilemma, and sequential games. Although the class informally assumes some knowledge about decision theory, it is an introductory class, and all of the decision theory parts are easy to understand, as it is not in-depth. There are no prerequisites for this class, nor is there a requirement for a solid mathematic background.
+
 ## Lectures
+
```
Comment (mikinty) [inline electives/80405.md]: If you're going to add the spaces to this header, please do it to the rest of the doc as well

**Review by mikinty — CHANGES_REQUESTED** [review]

**Comment by JCao-CMU** [comment]
Spacing is added for all headers. 
I never used markdown before, and all commits were done in the Github edit function. I have to copy and paste between Github edits and Grammarly...

**Comment by mikinty** [comment]
> Spacing is added for all headers. I never used markdown before, and all commits were done in the Github edit function. I have to copy and paste between Github edits and Grammarly...

I can tell you are using Github edits, but you can still edit the commit title and description to be more than just the default. In addition, you can make more substantial changes each time as well.

For the future I would suggest cloning this locally so that you can make changes accordingly there, with more freedom on version control and linting etc.

I'm actually going to close this PR because there are 8 commits for something that should be probably one commit. Can you please open a new PR and make new commits for the code? I know it's annoying but I don't want 8 commits in the repo if we don't need them.
